spotbugs/src/main/java/edu/umd/cs/findbugs/config/CommandLine.java

```diff
@@ -342,9 +354,35 @@ private int parse(String argv[], boolean dryRun) throws IOException, HelpRequest
             }
         }
 
+        if (!dryRun && !parsingStdin && parseStdinAsOptions) {
+            String[] stdinArgv = readStdinAsArgv();
+            if (0 != stdinArgv.length) {
+                parse(stdinArgv, true, dryRun);
+            }
+        }
+
         return arg;
     }
 
+    private String[] readStdinAsArgv() throws IOException {
+        List<String> stdinArgvList = new ArrayList<String>();
+
+        BufferedReader in = UTF8.bufferedReader(System.in);
+        while (true) {
+            String s = in.readLine();
+            if (s == null) {
+                break;
+            }
+            if (s.trim().isEmpty()) {
+                break;
+            }
+            stdinArgvList.add(s);
+        }
+        in.close();
```

Comment:
Use try-with-resources to close this `BufferedReader`.
(personally not so sure that we can close `System.in` or not.)

Reply:
I was also not sure if we can/should close System.in or not.  I copied this code from -auxclasspathFromInput in TextUICommandLine

Reply:
I went ahead and updated it with try-with-resources.